arrocke/gloss-translation
# feat: notes indication

## What has changed
- Added sticky note button next to words with notes 
- When clicking this sticky note button, the note tab is opened with focus on the translator editor.


## Connected Issues

closes #383 


## QA Steps


1. Select a word and add notes. Confirm that when the notes are saved, the note icon appears.
2. Close the sidebar
3. Press the sticky note. Confirm that the notes tab opens and the editor is focused.

## Post-Deployment
